src/shared/backend/replicon_client.rs

```diff
@@ -57,12 +57,6 @@ impl RepliconClient {
     /// Receives all available messages from the server over a channel.
     ///
     /// All messages will be drained.
-    ///
-    /// <div class="warning">
-    ///
-    /// Should only be called from the messaging backend.
```

Comment:
Why remove this?

Reply:
It's not called by messaging backends. It's called by Replicon to read the messages. Maybe I should even make it private :thinking:

Reply:
Yes make it private if it's not for the public or backend APIs.

Reply:
Done!